x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.tsx

```diff
@@ -599,20 +599,9 @@ export function DimensionEditor(props: DimensionEditorProps) {
   const customParamEditor = ParamEditor ? (
     <>
       <ParamEditor
-        layer={state.layers[layerId]}
-        layerId={layerId}
-        activeData={props.activeData}
+        {...paramEditorProps}
```

Comment:
While debugging some missing props, I've noticed this duplication here. I can be moved into a separate PR, but it's well isolated already.